client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -518,12 +538,26 @@ protected String getPath(Method method, Object[] arguments, JaxRsMeta meta) {
         return path;
     }
 
+    private List<Tuple<Function<Object, Object>, Annotation[]>> getBeanParamGetters(Class beanParamType) {
+        List<Tuple<Function<Object, Object>, Annotation[]>> result = new ArrayList<>();
+        for (Field field : beanParamType.getDeclaredFields()) {
+            Optional<Function<Object, Object>> getter = ReflectionUtil.getter(beanParamType, field.getName());
+            if (getter.isPresent()) {
+                result.add(new Tuple<>(
+                    getter.get(),
+                    field.getAnnotations()
+                ));
+            }
+        }
+        return result;
+    }
+
     protected String serialize(Object value) {
```

Comment:
is there a reason why we have protected methods? would it be safe to make them "package private" and not expose them outside of the package. Because i have a feeling that it might be what we intended? But might be risky to risky to do?

Reply:
Having them protected would be to allow subclasses to change behaviour. It is easy to make this new method protected in the future, but going the other way is not possible, so I think private is right atm.